package.json

```diff
@@ -24,6 +22,7 @@
     "@bcrs-shared-components/enums": "1.0.39",
     "@bcrs-shared-components/folio-number-input": "1.1.18",
     "@bcrs-shared-components/interfaces": "1.0.61",
+    "@bcrs-shared-components/mixins": "1.1.21",
```

Comment:
I'm adding this package as I will be using the `CountriesProvincesMixin` from here now. It was previously a local mixin (I've deleted it from the project).